src/virtualenv/create/via_global_ref/builtin/pypy/pypy3.py

```diff
@@ -44,6 +44,8 @@ def sources(cls, interpreter):
         host_lib = Path(interpreter.system_prefix) / "lib"
         if host_lib.exists() and host_lib.is_dir():
             for path in host_lib.iterdir():
+                if path.is_dir():
+                    continue
```

Comment:
Once upon  a time we needed to copy any shared object files in `base/lib` but these days we use RPATH in the `libpypy3-c.so` to find the `lib` dir, and that main shared object is now symlinked.

So in short, iterating over `base/lib` and copying the files is no longer needed, so lines 45-48 can be deleted. That means this whole method can be deleted.

Reply:
... as long as `os.symlink` is available where the "portable PyPy" downloads (the ones from https://www.pypy.org/download.html) are used. Linux and brew build PyPy against system libraries, so they do not need the shared objects in the `base/lib` directory .

Reply:
@mattip for 3.6 we still need this, not? What about Windows?

Reply:
No, 3.6 also uses symlinks and RPATH. Windows never needed this.

@mgorny could you change line 30, 31 to use `pypy` instead of `python` ?